omero/sysadmins/version-requirements.txt

```diff
@@ -320,16 +320,12 @@ Microsoft Windows
       - to Jan 2023
       - Supported
       - Recommended
-      - Recommended
+      - Unsupported
       - `Ref <http://support.microsoft.com/lifecycle/?p1=16526>`__
 
-XP is now unsupported upstream, and Server 2003 will be unsupported
-within the timeframe of 5.1 support. Vista is likely supported but
-usage is minimal and it is untested, hence marked as dropped. 7/Server
-2008 and 8/Server 2012 are both supported and used. Server 2008 builds
-in place; no 2012 builds or testing in place yet. No regular Windows
-CI deployment and testing is in place other than manually. Some manual
-testing on Windows 7 VMs and machines.
+Insufficient resources are available for regular CI deployment and testing so
+a decision has been made to drop support in favor of other priorities. See
+`this blog post for full details <http://blog.openmicroscopy.org/tech-issues/future-plans/deployment/2016/03/22/windows-support/>`_.
```

Comment:
I think a comment should be made about which Windows versions are supported by the clients here as well.  You might want to put a `†` after `Unsupported` after each line changed here, with a definition list below describing that client support will continue.